libqtile/widget/crypto_ticker.py

```diff
@@ -74,14 +82,36 @@ def __init__(self, **config):
         if self.symbol == "":
             self.symbol = "$"
 
+        if self.api in self.QUERY_URL_DICT:
+            self.query_url = self.QUERY_URL_DICT[self.api]
+        else:
+            apis = sorted(self.QUERY_URL_DICT.keys())
+            logger.error(
+                "%s is not a valid API. Use one of the list: %s.",
+                self.api,
+                str(apis),
```

Comment:
I think this `str` call is redundant because the `"%s"` in the format string will do it.

Reply:
That's correct. The `str` call was removed. Commit: 25096b15e9602a690d0649e79e195f7ecdce0ccc